CHANGELOG.md

```diff
@@ -4,6 +4,218 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.15.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.15.0-rc1) - 2021-07-15
+
+* BREAKING
+  * Make app.ini permissions more restrictive (#16266)
+  * Refactor Webhook + Add X-Hub-Signature (#16176)
+  * Add asymmetric JWT signing (#16010)
+  * Clean-up the settings hierarchy for issue_indexer queue (#16001)
+  * Change default queue settings to be low go-routines (#15964)
+  * Improve assets handler middleware (#15961)
+  * Rename StaticUrlPrefix to AssetUrlPrefix (#15779)
+  * Use a generic markup class to display externally rendered files and diffs (#15735)
+  * Add frontend testing, require node 12 (#15315)
+  * Add `/assets` as root dir of public files (#15219)
```

Comment:
This is misleading.
```suggestion
  * Move (custom) assets into subpath `/assets` (#15219)
```
Also the docs do not reflect this, as [noted on discord](https://discord.com/channels/322538954119184384/561007778139734027/864939872427311134)
(This PR in general was quite badly tested, I'd not be suprised if people stumble over more breakage 🙃)